docs/ci-reference.md

```diff
@@ -0,0 +1,31 @@
+# How to run integration test job in CI
+
+For default oc, use the configuration in .travis.yaml. For example 
+
+```sh
+  # Run main e2e tests
+    - <<: *base-test
+      stage: test
+      name: "Main e2e tests"
+      script:
+        - ./scripts/oc-cluster.sh
+        - make bin
+        - sudo cp odo /usr/bin
+        - oc login -u developer
+        - make test-main-e2e
+```
+
+Incase there is a need of running odo integration test on top of specific oc version, use env variable `OPENSHIFT_CLIENT_BINARY_URL` to pass the [released](https://github.com/openshift/origin/releases) oc client URL in `.travis.yaml`. For oc v3.10.0, use the configuration 
```

Comment:
`Incase there is a need of running odo integration test on top of specific oc version` -> `If the need presents itself to run odo integration tests against a specific version of Openshift`

Reply:
Done